utils/python/CIME/SystemTests/eri.py

```diff
@@ -90,6 +90,9 @@ def run_phase(self):
         clone1.set_value("REST_N", rest_n1)
         clone1.set_value("HIST_OPTION", "never")
         clone1.flush()
+        # if the initial case is hybrid this will put the reference data in the correct location
+        if orig_runtype == "hybrid":
```

Comment:
I have no idea; you may be right. My bigger point was that it seems unnecessary to have this conditional at all, and since it may be overly restrictive, I was thinking it could be deleted.